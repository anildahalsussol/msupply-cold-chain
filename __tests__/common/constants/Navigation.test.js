import { NAVIGATION } from '~constants';

describe('Constants: NAVIGATION', () => {
  it('Matches snapshot', () => {
    expect(NAVIGATION).toMatchSnapshot();
  });
});
