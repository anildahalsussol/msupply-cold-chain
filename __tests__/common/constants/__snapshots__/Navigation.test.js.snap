// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Constants: NAVIGATION Matches snapshot 1`] = `
Object {
  "CONFIG": Object {
    "SENSOR_STACK": Object {
      "GESTURE_DIRECTION": "horizontal",
      "HEADER_MODE": "none",
    },
  },
  "SCREENS": Object {
    "MAIN_TABS": Object {
      "MAP": "MAIN_TABS/Map",
      "SENSORS": "MAIN_TABS/Sensors",
      "SETTINGS": "MAIN_TABS/Settings",
    },
    "SENSOR_STACK": Object {
      "SENSOR_DETAIL": "SENSOR_STACK/SensorDetail",
      "TABS": "SENSOR_STACK/Tabs",
    },
    "SETTINGS_STACK": Object {
      "BLUETOOTH": "SETTINGS_STACK/Bluetooth",
      "DEVELOPER": "SETTINGS_STACK/Developer",
      "MENU": "SETTINGS_STACK/Menu",
      "SENSORS": "SETTINGS_STACK/Sensors",
      "SENSOR_DETAIL": "SETTINGS_STACK/SensorDetail",
      "TEMPERATURE_BREACH": "SETTINGS_STACK/TemperatureBreach",
      "TEMPERATURE_BREACH_DETAIL": "SETTINGS_STACK/TemperatureBreachDetail",
      "TEMPERATURE_CUMULATIVE_DETAIL": "SETTINGS_STACK/TemperatureCumulativeDetail",
    },
  },
}
`;
